MIGRATION-V6.md

```diff
@@ -0,0 +1,146 @@
+# Upgrading to ibm-watson@6.0
+
+_Note: If migrating from a version less than 5.0, also see the [v5 migration guide wiki](https://github.com/watson-developer-cloud/node-sdk/wiki/v5-Migration-Guide)._
+
+- [Breaking Changes](#breaking-changes)
+  - [Support for callbacks dropped](#support-for-callbacks-dropped)
+  - [Support for Node v10 dropped](#support-for-node-v10-dropped)
+  - [Breaking changes by service](#breaking-changes-by-service)
+
+- [New Features by Service](#new-features-by-service)
+
+## Breaking changes
+
+#### Promises
+The detailed response is always returned. The `return_response` parameter is removed.
+```js
+const response = await listWorkspaces();
+console.log(response.result); // prints the body
+console.log(response.headers); // prints the headers
+```
```

Comment:
Didn't this happen in the last major release? We don't need to document it here if so